pkg/scheduler/preemption/preemption.go

```diff
@@ -579,3 +590,12 @@ func quotaReservationTime(wl *kueue.Workload, now time.Time) time.Time {
 	}
 	return cond.LastTransitionTime.Time
 }
+
+// getWorkloadInfosFromTargets extracts workload.Info from Target slice for logging
+func getWorkloadInfosFromTargets(targets []*Target) []*workload.Info {
+	workloads := make([]*workload.Info, len(targets))
+	for i, target := range targets {
+		workloads[i] = target.WorkloadInfo
+	}
+	return workloads
+}
```

Comment:
```suggestion
func GetWorkloadReferences(targets []*preemption.Target) []klog.ObjectRef {
	return slices.Map(targets, func(t **preemption.Target) klog.ObjectRef { return klog.KObj((*t).WorkloadInfo.Obj) })
}
```

We can remove https://github.com/kubernetes-sigs/kueue/blob/0666f2ceb937a8f2d9198e381f9dd5cd19445d1a/pkg/scheduler/logging.go#L52-L54, then reuse new `GetWorkloadReferences()` everywhere.